src/entry.js

```diff
@@ -1,3 +1,5 @@
+require('src/assets/stylesheets/base.css');
```

Comment:
@j-clark Is this correct for your config, or should it be added to the  contexts?